### PR TITLE
Magento_Shipping: avoid using deprecated escape* methods from Abstrac…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests
I have a feeling that static tests will be failing here

File: app/code/Magento/Shipping/view/adminhtml/templates/create/items.phtml [hunk 1]
```diff
@@ -96,7 +99,7 @@
                        type="checkbox"/>
                 <label class="admin__field-label"
                        for="notify_customer">
-                    <span><?=$block->escapeHtml(__('Append Comments')) ?></span></label>
+                    <span><?=$escaper->escapeHtml(__('Append Comments')) ?></span></label>
```
Comment (ihor-sviziev) [inline app/code/Magento/Shipping/view/adminhtml/templates/create/items.phtml:102]:
```suggestion
                    <span><?= $escaper->escapeHtml(__('Append Comments')) ?></span></label>
```

**Review by ihor-sviziev — APPROVED** [review]

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE,Functional Tests EE

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE,Functional Tests EE

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests CE,Functional Tests EE

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8643](https://jira.corp.magento.com/browse/ENGCOM-8643) has been created to process this Pull Request

**Pull request closed without merging**